app/helpers/blob_helper.rb

```diff
@@ -1,10 +1,18 @@
 module BlobHelper
-  def highlightjs_class(blob_name)
-    if no_highlight_files.include?(blob_name.downcase)
-      'no-highlight'
-    else
-      blob_name.downcase
+  def highlight(blob_name, blob_content, wrap = true)
```

Comment:
Unused method argument - `wrap`. If it's necessary, use `_` or `_wrap` as an argument name to indicate that it won't be used.